frontend/src/lib/components/PropertyFilters/OperatorValueSelect.tsx

```diff
@@ -83,13 +83,14 @@ export function OperatorSelect({ operator, operators, onChange }: OperatorSelect
                 label: operatorMap[operator || 'exact'],
             }}
             placeholder="Property key"
-            onChange={(_, newOperator) => {
-                onChange(newOperator.value as PropertyOperator)
+            onChange={(_value, op) => {
+                const newOperator = op as { value: PropertyOperator } & Record<string, any>
```

Comment:
Why the `Record<string, any>`? Are there any other components that are not properly typed that this could be hiding?